### PR TITLE
pass build compilers to TBB

### PR DESCRIPTION
**Pull request by ghost**
Use the same compilers used to build the rest of SonarW to build TBB

### CODE REVIEW

File: SonarExternal.cmake [hunk 1]
```diff
@@ -1310,8 +1310,8 @@ function(build_tbb)
     CONFIGURE_COMMAND ""
     BUILD_IN_SOURCE 1
     BUILD_COMMAND make
-      CC=gcc
-      CXX=g++
+      CC=${CMAKE_C_COMPILER}
+      CXX=${CMAKE_CXX_COMPILER}
```
Comment (yhagerjs) [inline SonarExternal.cmake]:
```suggestion
      CC=${CMAKE_C_COMPILER_LAUNCHER}\ ${CMAKE_C_COMPILER}
      CXX=${CMAKE_CXX_COMPILER_LAUNCHER}\ ${CMAKE_CXX_COMPILER}
```

Comment (yhagerjs) [reply SonarExternal.cmake]: sorry, I don't know how to do muti-line suggestion, but you get the point

**Comment by mklajs** [comment]
I'm still thinking through how best to handle this. The basic issue is that launcher is a semicolon delimited list, in general, this will contain spaces after expansion, and working out the correct escaping to pass this through make and shell is a yak I don't yet worked out how to shave.

**Comment by yhagerjs** [comment]
Don't overthink it. The launcher is `ccache` most of the time. We're not using lists, so just do the simplest thing I suggested above (escape the space between the launcher and the compiler)

**Comment by yhagerjs** [comment]
@mklajs, can you add the launcher so we can merge this?

**Pull request closed without merging**